tests/model/test_ocrd_page.py

```diff
@@ -1,4 +1,8 @@
-from tests.base import TestCase, main, assets, create_ocrd_file_with_defaults
+# -*- coding: utf-8 -*-
```

Comment:
UTF-8 source encoding is the [default in Python 3](https://www.python.org/dev/peps/pep-3120/), so this is not necessary anymore.